contracts/aevm.md

```diff
@@ -2,18 +2,43 @@
 
 The AEVM is a version of the EVM: https://github.com/ethereum/yellowpaper
 
-The AEVM is emulated inside the Ethereum node an no marshaling of
+The AEVM can be emulated inside the Æternity node an no marshaling of
 arguments, code and data is necessary.  This makes for fast upstart
 and faster execution of most smaller contracts.
 
 There are a few differences between the EVM and the AEVM.
 
+The AEVM provides an number of primitive operations that can be
+used to interact with the chain.
+
 The SELFDESTRUCT instruction is not immediate, instead the contract is
 set in a disabled state where neither new contracts nor call
 transactions can call the contract directly. When all other contracts
 referring to a disabled contract are fully disabled the contract is
 disabled.
 
+## Æternity primitive operations
+
+Interaction with the first class objects of the Æternity chain (e.g.
+oracles, names, and state channels) is done through calls to a
+built in contract at address 0.
+
+The first argument in the call specifies witch primop to call.
+The following arguments are encoded as Sophia data.
+
+| OpNo | Name                |             Arguments | Return value    |
+| ---- | ------------------- | --------------------- | --------------- |
+|    1 | Spend               | `Recipient : address` | `Nil`           |
```

Comment:
I see opno in line with [code](https://github.com/aeternity/epoch/blob/3e5e28a1452e8b7e17af9f940381c72b8d690d4d/apps/aebytecode/include/aeb_opcodes.hrl#L151-L162). Ok.